effort_controllers/test/test_joint_group_effort_controller.cpp

```diff
@@ -18,18 +18,28 @@
 #include <memory>
 #include <string>
 #include <vector>
+#include <utility>
 
-#include "gtest/gtest.h"
-
+#include "hardware_interface/loaned_command_interface.hpp"
 #include "hardware_interface/types/hardware_interface_return_values.hpp"
-#include "hardware_interface/joint_handle.hpp"
 #include "lifecycle_msgs/msg/state.hpp"
 #include "rclcpp/utilities.hpp"
 #include "rclcpp_lifecycle/node_interfaces/lifecycle_node_interface.hpp"
 #include "test_joint_group_effort_controller.hpp"
-#include "test_robot_hardware/test_robot_hardware.hpp"
 
 using CallbackReturn = forward_command_controller::ForwardCommandController::CallbackReturn;
+using hardware_interface::LoanedCommandInterface;
+
+namespace
+{
+rclcpp::WaitResultKind wait_for(rclcpp::SubscriptionBase::SharedPtr subscription)
+{
+  rclcpp::WaitSet wait_set;
+  wait_set.add_subscription(subscription);
+  const auto timeout = std::chrono::seconds(10);
+  return wait_set.wait(timeout).kind();
+}
+}
```

Comment:
This is also often used snippet when testing controllers. Should we add it into ȧ `test_resources` package?